api/lib/domain/services/certification-challenges-service.js

```diff
@@ -4,16 +4,18 @@ const CertificationChallenge = require('../models/CertificationChallenge');
 module.exports = {
 
   generateCertificationChallenges(userCompetences, certificationCourseId) {
-    return _.flatMap(userCompetences, (userCompetence) => {
-      return _.map(userCompetence.challenges, (challenge) => {
-        return new CertificationChallenge({
-          challengeId: challenge.id,
-          competenceId: challenge.competenceId,
-          associatedSkillName: challenge.testedSkill.name,
-          associatedSkillId: challenge.testedSkill.id,
-          courseId: certificationCourseId,
-        });
+    const challenges = _.flatMap(userCompetences, (userCompetence) => {
+      return userCompetence.challenges;
+    });
```

Comment:
Je crois qu'on peut l'écrire aussi comme ça 
```suggestion
    const challenges = _(userCompetences).flatMap(uc => uc.challenges)
```
ou encore
```suggestion
   const challenges = _.flatMap(userCompetences, (uc) => uc.challenges);
```